.github/workflows/checkstyle.yml

```diff
@@ -0,0 +1,19 @@
+name: Checkstyle
+
+on:
+  push:
+    branches: main
```

Comment:
I think this is supposed to be an array.

Reply:
Looking at the [syntax guide](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions), I'm not sure if an array is strictly required when there's only one entry there but it probably wouldn't hurt either...